views/default/forms/admin/site/settings/users.php

```diff
@@ -32,18 +32,15 @@
 }
 
 $result .= elgg_view_field([
-	'#type' => 'select',
+	'#type' => 'checkbox',
 	'#label' => elgg_echo('installation:adminvalidation:notification:label'),
 	'#help' => elgg_echo('installation:adminvalidation:notification:description'),
 	'#class' => $classes,
 	'name' => 'admin_validation_notification',
-	'value' => elgg_get_config('admin_validation_notification'),
-	'options_values' => [
-		'' => elgg_echo('option:no'),
-		'direct' => elgg_echo('installation:adminvalidation:notification:direct'),
-		'daily' => elgg_echo('interval:daily'),
-		'weekly' => elgg_echo('interval:weekly'),
-	],
+	'checked' => (bool) elgg_get_config('admin_validation_notification'),
+	'switch' => true,
+	'value' => 1,
+	'default' => false,
```

Comment:
why have a default of `false` the default already is `0`

Reply:
leaving this for now